.github/workflows/docker.yaml

```diff
@@ -0,0 +1,80 @@
+name: Docker
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "v*.*.*"
```

Comment:
Just checking, does this covers RC tags like `v0.1.2-rc.3` as well?

Reply:
I just took it from: https://github.com/osmosis-labs/mesh-security-sdk/blob/main/.github/workflows/release.yml#L6
I would like to have something like: 
https://github.com/cosmos/cosmos-sdk/blob/main/.github/workflows/docker.yml#L11...L13

But did not want to deviate from the repo practices